src/components/tutorials/basic-tutorial.js

```diff
@@ -14,23 +14,23 @@ class BasicTutorial extends React.Component {
       <div>
         <div>
           <div className='temp-grid'>
-            <h1 className='ds-h1'>API Tutorial</h1>
-            <p>The Submissions API is an easy way to manage your performance data with CMS. Performance data is organized into <em>submissions</em>, which can have many <em>measurements</em>. Measurements within a submission are also grouped by category (e.g. Improvement Activities) and submission method (e.g. CMS web interface) into <em>measurement sets</em>.</p>
-            <p>Since the API and scoring calculations are still being finalized, please note that some numbers (especially scoring) may be inaccurate. The <a href='https://qpp-submissions-sandbox.navapbc.com/'>Interactive Docs</a> will always reflect the latest design.</p>
-            <p>Let's walk through an example of how we might submit performance data!</p>
-            <h2 className='ds-h2' id='creating-a-submission'>
+            <h1 className='ds-h1'>Tutorial: Create and Score Data</h1>
+            <p>The Submissions API is an easy way to manage your performance data with CMS. Performance data is organized into submissions, which can have many measurements. Measurements within a submission are also grouped by category (e.g. Quality) and submission method (e.g. registry) into measurement sets.</p>
+            <p>Since the API and scoring logic are updated on a regular basis as per the annual publication of the QPP Final Rule, please note that some numbers (especially scoring) may be inaccurate. The <a href='https://qpp-submissions-sandbox.navapbc.com/'>Interactive Docs</a> and Developer Preview will always reflect the latest design.</p>
```

Comment:
Please update this paragraph to:

<p>Since the API and scoring logic are updated on a regular basis as per the annual publication of the QPP Final Rule, please note that some numbers (especially scoring) in the tutorial below may be inaccurate. The <a href='https://qpp-submissions-sandbox.navapbc.com/'>Interactive Docs</a> and Developer Preview will always reflect the latest design.</p>

Reply:
^^ I just added clarification that numbers may be inaccurate in the tutorial example.